src/System.Devices.I2c/Provider/ProviderI2cBusSpeed.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Devices.I2c.Provider
+{
+    /// <summary>
+    /// Describes the bus speeds that are available for connecting to an inter-integrated
+    /// circuit (I²C) device. The bus speed is the frequency at which to
+    /// clock the I²C bus when accessing the device.
+    /// </summary>
+    public enum ProviderI2cBusSpeed
```

Comment:
Why do we need two enums (and other types) for the client and the provider? They seem to share values.

Reply:
Good point.  I agree and was thinking there could eventually be common enums.  In addition, I'm still scratching my head a little on functionality of Providers and planned that topic to be a future investigation.  Just thinking it would be good to go ahead for something to start with.  Until a review (and better strategy) how Providers work... I went ahead and removed the related files for now.